pytx3/pytx3/signal_type/signal_base.py

```diff
@@ -52,6 +52,14 @@ def process_descriptor(self, descriptor: t.Dict[str, t.Any]) -> bool:
         """
         return False
 
+    def process_indicator(self, indicator: t.Dict[int, t.Any]) -> bool:
+        """
+        Add ThreatIndicator to the state of this type, if it is for this type.
+
+        Return true if the indicator was used.
+        """
+        return False
+
```

Comment:
What's this used for?

Reply:
Place holder for the super class - if a child class doesn't implement process_indicator it defaults to false.